### DISCUSSION TITLE
Is "Data too long for column RequestBody" to do with ServiceStack MySQLConnector?

### DISCUSSION THREAD

**Discussion by CharlieArmstrongDev** [ServiceStack/Discuss#28] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/97681489/186680118-a9deaa6a-fc69-4101-aace-2831078c7a3e.png)
I am trying to use one of my AutoQuery endpoints generated from a DTO in ServiceModel.

As you can see, it is claiming the data is too long - Is this to do with ServiceStack trying to put it into the CrudEvents table or something? This length for a request (anything over 255 characters) doesn't seem unreasonable at all so I'd be surprised if no-one had run into this problem before - is there an obvious solution to this? Should I disable the CrudEvents table entirely somehow?

**Comment by CharlieArmstrongDev** [comment]
Although there is the possibility it is to do with something else behind the scenes in how ServiceStack handles these things, as it mentions Row 1, while crudevents already has many entries.

**Comment by mythz** [comment, marked as answer]
It'll be because it exceeds MySql's VARCHAR string column, I've updated RequestBody to use a text column in [this commit](https://github.com/ServiceStack/ServiceStack/commit/ebb23e08881350735cfd78fddd9c60ebe468b232). 

If you change the `CrudEvent.RequestBody` column to **LONGTEXT** it should resolve this issue.
